### PR TITLE
[Feature]Add sign out

### PR DESCRIPTION
**Pull request by jigsawye**
For convenience, please see the commit messages.

This branch is dependence on https://github.com/inwinstack/s3-portal-ui/pull/30, please merge this PR after https://github.com/inwinstack/s3-portal-ui/pull/30 are merged.

### CODE REVIEW

File: src/styles/dialog.css [hunk 1]
```diff
@@ -3,8 +3,8 @@
  * @author Jamie jamie.h@inwinstack.com
  */
 
-.md-confirm-dialog-warn-theme md-dialog-actions button:last-child {
-  color: #FF6D00;
+.md-default-theme md-dialog-actions button:last-child {
```
Comment (JamieHo3312) [inline src/styles/dialog.css]: The `.md-confirm-dialog-warn-theme` should be reserved here.
[As the discussion before](https://github.com/inwinstack/s3-portal-ui/pull/21), you have to add `.theme('confirm-dialog-warn')` in JavaScript to highlight the button, please refer the confirm dialog of [CodePen](http://codepen.io/jamie3312/pen/dMzogN?editors=1010). This change should be revert, thanks.

Comment (jigsawye) [inline src/styles/dialog.css]: If I use the `theme()` method, the browser console will has some warning message when I clicked the button to open confirm dialog:

```
attempted to use unregistered theme 'confirm-dialog-warn'
Attempted to use unregistered theme 'confirm-dialog-warn'. Register it with $mdThemingProvider.theme().
Attempted to use unregistered theme 'confirm-dialog-warn'. Register it with $mdThemingProvider.theme().
attempted to use unregistered theme 'confirm-dialog-warn'
Attempted to use unregistered theme 'confirm-dialog-warn'. Register it with $mdThemingProvider.theme().
<md-backdrop> may not work properly in a scrolled, static-positioned parent container.
```

In the [codepen example](http://codepen.io/jamie3312/pen/dMzogN?editors=1010) also has the issue. Do you have any idea to resolve the issue?

Comment (JamieHo3312) [inline src/styles/dialog.css]: I think we use an incorrect method to the theme. To avoid getting stuck of the project progress, please revert this change and remove the `.theme()` until we find out the correct usage of the theme.
